index.md

```diff
@@ -11,10 +11,11 @@ Governing Documents:
 
 * [Antitrust Policy](./antitrust.md)
 * [Code of Conduct](./code-of-conduct.md)
-* [Project Incubation Exit Criteria](./project-incubation-exit.md)
-* [Project Lifecycle](./project-lifecycle.md)
-* [Release Taxonomy](./release-taxonomy.md)
+* [Project Governance](projects)
```

Comment:
It feels like a better folder name would be `governance`

Reply:
Also, are the Code of Conduct and the Antitrust Policy not part of the governance of Hyperledger?

Reply:
I was planning to create a page here for the new project incubation. With that it bloated the root directory. This made me consider anything that defines project (viz, such as project lifecycle, project incubation exit criteria, releases etc) move into its own folder.

Let me know your thoughts on better folder organization.